code/web/src/modules/user/ProfilePicture.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { File } from '../../ui/input'
+import { upload, messageShow, messageHide } from '../common/api/actions'
+import { renderIf } from '../../setup/helpers'
+import { routeImage } from '../../setup/routes'
+import { APP_URL, APP_URL_API } from '../../setup/config/env'
+import { updateUser } from './api/actions'
+
+class ProfilePicture extends Component {
+  constructor() {
+    super() 
+    this.state = {
+      imgURL: '',
+      name: '',
```

Comment:
Great work with this component!! So glad we can get images uploaded.

One thought on this component- since the user isn't updating their name/email in this component, I don't think we need these properties in `state` or our `componentDidMount`. Then, in our `profileState` on line 83, the only part of Redux store we need to connect to this component `user.image`. From my understanding, we should connect as little as possible to Redux store as to not pass around app data that we don't have to.

I don't think we need to update these changes before approving the PR! And this feedback might change depending on how we merge @leighlars component into this one.